cmake/schemes/url_sha1_boost.cmake.in

```diff
@@ -88,8 +89,9 @@ else()
 endif()
 
 if("@MSVC@" OR "@MINGW@")
```

Comment:
My practice of hacking Boost build scheme (and Hunter in general) told me not to change anything that doesn't produce bugs. So let's focus on MinGW compiler + Windows platform until somebody report a problem about Visual Studio.